mne_bids/write.py

```diff
@@ -951,12 +937,13 @@ def _write_raw_edf(raw, bids_fname):
     raw.export(bids_fname)
 
 
+@fill_doc
 def make_dataset_description(path, name, data_license=None,
```

Comment:
I don't see any global vars used by this function, so I don't see why it would fail with the `verbose` decorator

Reply:
It's the BIDSVersion which is checked in the test